[CI] formatting with black 26.1.0

`black` `26.1.0` introduced several changes in formatting which trigger some errors:

```diff
--- /home/runner/work/tox-passenv/tox-passenv/src/tox_passenv/_tox4_plugin.py	2026-01-19 07:35:24.934514+00:00
would reformat /home/runner/work/tox-passenv/tox-passenv/src/tox_passenv/_tox4_plugin.py
+++ /home/runner/work/tox-passenv/tox-passenv/src/tox_passenv/_tox4_plugin.py	2026-01-19 07:35:36.589279+00:00
@@ -1,10 +1,9 @@
 import logging
 import os
 
 from tox.plugin import impl
-
 
 logger = logging.getLogger(__name__)
 
 
 @impl
/home/runner/work/tox-passenv/tox-passenv/tests/conftest.py already well formatted, good job.
--- /home/runner/work/tox-passenv/tox-passenv/tests/test_plugin.py	2026-01-19 07:35:24.934514+00:00
+++ /home/runner/work/tox-passenv/tox-passenv/tests/test_plugin.py	2026-01-19 07:35:36.709635+00:00
@@ -62,13 +62,11 @@
             f'print("{var}:", os.environ.get("{var}", "NOT_SET"));'
         )
 
     command_template = f"python -c 'import os;{env_vars_commands}'"
 
-    project.contents[
-        "tox.ini"
-    ] = f"""\
+    project.contents["tox.ini"] = f"""\
         [tox]
         env_list = {env_name}
         [testenv]
         {option} = {pass_name}
         commands =
@@ -143,13 +141,11 @@
             f'print("{var}:", os.environ.get("{var}", "NOT_SET"));'
         )
 
     command_template = f"python -c 'import os;{env_vars_commands}'"
 
-    project.contents[
-        "tox.ini"
-    ] = f"""\
+    project.contents["tox.ini"] = f"""\
         [tox]
         env_list = {env_name}
         [testenv]
         {option} = {pass_name}
         commands =
@@ -194,13 +190,11 @@
would reformat /home/runner/work/tox-passenv/tox-passenv/tests/test_plugin.py
             f'print("{var}:", os.environ.get("{var}", "NOT_SET"));'
         )
 
     command_template = f"python -c 'import os;{env_vars_commands}'"
 
-    project.contents[
-        "tox.ini"
-    ] = f"""\
+    project.contents["tox.ini"] = f"""\
         [tox]
         env_list = {env_name}
         [testenv]
         skip_install = True
         commands =

Oh no! 💥 💔 💥
2 files would be reformatted, 5 files would be left unchanged.
Error: Process completed with exit code 1.
```

See for details:
https://black.readthedocs.io/en/stable/change_log.html

--- a/.github/workflows/pipelines.yml
+++ b/.github/workflows/pipelines.yml
@@ -16,7 +16,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: ['3.9', '3.10', '3.11', '3.12', '3.13', '3.14']
+        python-version: ['3.10', '3.11', '3.12', '3.13', '3.14']
 
     steps:
     - uses: actions/checkout@v5
@@ -49,7 +49,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: ['3.9', '3.10', '3.11', '3.12', '3.13', '3.14']
+        python-version: ['3.10', '3.11', '3.12', '3.13', '3.14']
 
     steps:
     - uses: actions/checkout@v5
@@ -79,7 +79,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: ['3.9', '3.10', '3.11', '3.12', '3.13', '3.14']
+        python-version: ['3.10', '3.11', '3.12', '3.13', '3.14']
 
     steps:
     - uses: actions/checkout@v5

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -5,7 +5,7 @@ build-backend = "setuptools.build_meta"
 [project]
 name = "tox-passenv"
 description = "Additional environment variable names to pass into tox test env"
-requires-python = ">= 3.9"
+requires-python = ">= 3.10"
 dependencies = [
   "tox >= 3",
 ]
@@ -30,7 +30,6 @@ classifiers = [
   "Topic :: Utilities",
   "Programming Language :: Python :: 3",
   "Programming Language :: Python :: 3 :: Only",
-  "Programming Language :: Python :: 3.9",
   "Programming Language :: Python :: 3.10",
   "Programming Language :: Python :: 3.11",
   "Programming Language :: Python :: 3.12",

--- a/src/tox_passenv/_tox4_plugin.py
+++ b/src/tox_passenv/_tox4_plugin.py
@@ -3,7 +3,6 @@ import os
 
 from tox.plugin import impl
 
-
 logger = logging.getLogger(__name__)
 
 

--- a/tests/test_plugin.py
+++ b/tests/test_plugin.py
@@ -64,9 +64,7 @@ def test_plugin_usage(tox_project, monkeypatch, passenv_data):
 
     command_template = f"python -c 'import os;{env_vars_commands}'"
 
-    project.contents[
-        "tox.ini"
-    ] = f"""\
+    project.contents["tox.ini"] = f"""\
         [tox]
         env_list = {env_name}
         [testenv]
@@ -145,9 +143,7 @@ def test_no_plugin_usage(tox_project, monkeypatch, passenv_data):
 
     command_template = f"python -c 'import os;{env_vars_commands}'"
 
-    project.contents[
-        "tox.ini"
-    ] = f"""\
+    project.contents["tox.ini"] = f"""\
         [tox]
         env_list = {env_name}
         [testenv]
@@ -196,9 +192,7 @@ def test_no_plugin_usage_tox3(tox_project, monkeypatch):
 
     command_template = f"python -c 'import os;{env_vars_commands}'"
 
-    project.contents[
-        "tox.ini"
-    ] = f"""\
+    project.contents["tox.ini"] = f"""\
         [tox]
         env_list = {env_name}
         [testenv]

